oss-internship-2020/libtiff/test/long_tag.cc

```diff
@@ -0,0 +1,137 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <array>
+
+#include "check_tag.h"
+#include "tiffio.h"  // NOLINT(build/include)
+
+// sapi functions:
+//  TIFFWriteScanline
+//  TIFFOpen
+//  TIFFClose
+//  TIFFGetField (from check_tag.c)
+//  TIFFSetField
+
+namespace {
+
+struct LongTag {
+  ttag_t tag;
+  short count;
+  unsigned value;
+};
+
+constexpr std::array<LongTag, 1> kLongTags = {
+    {TIFFTAG_SUBFILETYPE, 1,
+     FILETYPE_REDUCEDIMAGE | FILETYPE_PAGE | FILETYPE_MASK}};
+#define SPP 3  // kSamplePerPixel
```

Comment:
Why is this a define?

Reply:
because it is used for std::array<uint8_t, SPP>

Reply:
Please make this also a `constexpr`